config.js

```diff
@@ -6,13 +6,21 @@ const pkg = require('./package.json')
 
 const config = new Config([
   Config
-    .string('pipeline-url')
+    .string('ingestion-url')
     .required()
-    .desc('Pipeline Ingestion URL')
+    .desc('This can be the URL of a Log Analysis Ingestion endpoint, or a URL to '
+      + 'ingest directly into a Pipeline Source')
 , Config
-    .string('pipeline-key')
+    .string('ingestion-key')
     .required()
-    .desc('Your pipeline http source node key')
+    .desc('This can be an ingestion key directly to Log Analysis, or an ingestion key '
+      + 'for a Pipeline Source')
```

Comment:
maybe  - `This can be a standard ingestion key, or an ingestion key associated with a pipeline source`